conan/cli/commands/new.py

```diff
@@ -68,7 +68,7 @@ def get_template_vars():
                 template_vars.extend(meta.find_undeclared_variables(ast))
 
             injected_vars = {"conan_version", "package_name", "as_name"}
-            optional_vars = {"requires", "tool_requires"}
+            optional_vars = {"requires", "tool_requires", "outputRootDir"}
```

Comment:
Can we better use _snake_case_ for the argument instead of _camelCase_? It reads a bit strange with the other arguments around being _snake_case_

Reply:
Yes, sure! Sorry, I missed this